build/Targets/GenerateCompilerInternals.targets

```diff
@@ -45,125 +46,99 @@
+      <GeneratedSyntaxModelGetText>@(SyntaxGetTextDefinition -&gt; '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')</GeneratedSyntaxModelGetText>
     </PropertyGroup>
 
     <Delete Files="$(GeneratedSyntaxModelGetText)" />
   </Target>
 
-  <Target
-    Name="GenerateSyntaxModelTests"
-    Inputs="@(SyntaxTestDefinition);$(VBSyntaxGeneratorToolPath);$(CSharpSyntaxGeneratorToolPath)"
-    Outputs="@(SyntaxTestDefinition -> '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')"
-    Condition="'@(SyntaxTestDefinition)' != '' AND '$(BuildingProject)' == 'true' AND ('$(Language)' == 'VB' or '$(Language)' == 'C#')"
-  >
+  <Target Name="GenerateSyntaxModelTests"
+          Inputs="@(SyntaxTestDefinition);$(VBSyntaxGeneratorToolPath);$(CSharpSyntaxGeneratorToolPath)"
+          Outputs="@(SyntaxTestDefinition -&gt; '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')"
+          Condition="'@(SyntaxTestDefinition)' != '' AND '$(BuildingProject)' == 'true' AND ('$(Language)' == 'VB' OR '$(Language)' == 'C#')">
     <PropertyGroup>
       <SyntaxGenerator Condition="'$(Language)' == 'VB'">"$(CoreRunExe)" "$(VBSyntaxGeneratorToolPath)"</SyntaxGenerator>
       <SyntaxGenerator Condition="'$(Language)' == 'C#'">"$(CoreRunExe)" "$(CSharpSyntaxGeneratorToolPath)"</SyntaxGenerator>
-      <GeneratedSyntaxModelTests>@(SyntaxTestDefinition -> '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')</GeneratedSyntaxModelTests>
+      <GeneratedSyntaxModelTests>@(SyntaxTestDefinition -&gt; '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')</GeneratedSyntaxModelTests>
     </PropertyGroup>
 
-    <Exec
-      Command='$(SyntaxGenerator) &quot;@(SyntaxTestDefinition)&quot; &quot;$(GeneratedSyntaxModelTests)&quot; /test'
-      Outputs="$(GeneratedSyntaxModelTests)"
-    >
+    <Exec Command="$(SyntaxGenerator) &quot;@(SyntaxTestDefinition)&quot; &quot;$(GeneratedSyntaxModelTests)&quot; /test"
+          Outputs="$(GeneratedSyntaxModelTests)">
       <Output TaskParameter="Outputs" ItemName="FileWrites" />
     </Exec>
 
   </Target>
 
-  <Target
-    Name="CleanSyntaxModelTests"
-    Condition="'$(Language)' == 'VB' or '$(Language)' == 'C#'"
-  >
+  <Target Name="CleanSyntaxModelTests" Condition="'$(Language)' == 'VB' OR '$(Language)' == 'C#'">
     <PropertyGroup>
-      <GeneratedSyntaxModelTests>@(SyntaxTestDefinition -> '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')</GeneratedSyntaxModelTests>
+      <GeneratedSyntaxModelTests>@(SyntaxTestDefinition -&gt; '$(IntermediateOutputPath)\%(FileName)%(Extension).Generated$(DefaultLanguageSourceExtension)')</GeneratedSyntaxModelTests>
```

Comment:
I've reverted it back to `->` for readability and confirmed that it doesn't "violate" the xml spec.

The actual spec (https://www.w3.org/TR/REC-xml, see 2.4) only says:
>The ampersand character (&) and the left angle bracket (<) must not appear in their literal form, except when used as markup delimiters, or within a comment, a processing instruction, or a CDATA section. If they are needed elsewhere, they must be escaped using either numeric character references or the strings " &amp; " and " &lt; " respectively. The right angle bracket (>) may be represented using the string " &gt; ", and must, for compatibility, be escaped using either " &gt; " or a character reference when it appears in the string " ]]> " in content, when that string is not marking the end of a CDATA section.

So it is valid to do `->` in this context.